src/components/Label/Label.tsx

```diff
@@ -1,11 +1,12 @@
 import styled from 'styled-components';
 import { compose, margin, MarginProps, ResponsiveValue, variant } from 'styled-system';
+import { ComponentPropsWithoutRef } from 'react';
 import { Text } from '../Text/Text';
 import { Colors } from '../../essentials';
 import { theme } from '../../essentials/theme';
 import { get } from '../../utils/themeGet';
 
-interface LabelProps extends MarginProps {
+interface LabelProps extends ComponentPropsWithoutRef<'span'>, MarginProps {
```

Comment:
shouldn't it be `'span'`?

Reply:
@nlopin: That's what the associated test attempts to do. 
```
it('renders an <span> tag by default', () => {
    expect(render(<Label />)).toMatchHtmlTag('span');
});
```

Reply:
Got it 👍